rector.php

```diff
@@ -58,4 +59,6 @@
             // property can't be readonly as it's returned by reference and may be updated
             __DIR__ . '/src/TestFramework/Coverage/TestLocations.php',
         ],
+        ReadOnlyClassRector::class,
+        ReadOnlyPropertyRector::class,
```

Comment:
is it because it produces too many changes for now? Or is it because we don't these rules at all?

Reply:
It's out of scope of this ticket so I decided to exclude these rules for now